edge-src/common/PageUtils.js

```diff
@@ -166,6 +166,16 @@ export class SitemapResponseBuilder extends ResponseBuilder {
     return 'text/xml';
   }
 
+  _escapeXML(str) {
```

Comment:
We already have a similar function escapeHtml() at common-src/StringUtils.js

could you please use escapeHtml() instead of creating a duplicate function?

Thanks!

Reply:
Right missed that, rectified.